__tests__/app.test.js

```diff
@@ -35,3 +35,30 @@ describe("/api", () => {
         });
     });
 });
+describe("/api/articles/:article_id", () => {
+    test("GET: 200 sends an article object for the specified article ID", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .then(({ body }) => {
+            const { article } = body;
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
```

Comment:
Would you be expecting any number here or the actual article_id that has been requested?